benchmarks/src/main/scala/json/JsonBenchmark.scala

```diff
@@ -20,9 +22,18 @@ object Reference {
   implicit val mongoFormat: MongoFormat[Reference] = mongoProduct(apply _)
 }
 
-case class Price(id: String, value: BaseMoney)
+case class Price(id: String, value: BaseMoney, validUntil: DateTime)
 
 object Price {
+  // the lib does not ship a `MongoFormat[DateTime]`
+  implicit val dateTimeAsIsoStringFormat: MongoFormat[DateTime] = new MongoFormat[DateTime] {
```

Comment:
The choice of this instance dictates the `MongoFormat` results, I tried to pick something realistic.